frontend/src/app/main/data/comments.cljs

```diff
@@ -338,3 +409,43 @@
 
       (= :yours mode)
       (filter #(contains? (:participants %) (:id profile))))))
+
+(defn update-comment-thread-frame
+    ([thread ]
+   (update-comment-thread-frame thread uuid/zero))
+
+   ([thread frame-id]
+    (us/assert ::comment-thread thread)
+    (ptk/reify ::update-comment-thread-frame
+      ptk/UpdateEvent
+      (update [_ state]
+        (let [thread-id (:id thread)]
+          (assoc-in state [:comment-threads thread-id :frame-id] frame-id)))
+
+      ptk/WatchEvent
+      (watch [_ _ _]
+        (let [thread-id (:id thread)]
+          (->> (rp/mutation :update-comment-thread-frame {:id thread-id  :frame-id frame-id})
+               (rx/catch #(rx/throw {:type :update-comment-thread-frame}))
+               (rx/ignore)))))))
+
+(defn detach-comment-thread
+  "Detach comment threads that are inside a frame when that frame is deleted"
+  [ids]
+  (us/verify (s/coll-of uuid?) ids)
+
+  (ptk/reify ::detach-comment-thread
+    ptk/WatchEvent
+    (watch [_ state _]
+      (let [objects (wsh/lookup-page-objects state)
+            is-frame? (fn [id] (= :frame (get-in objects [id :type])))
+            frame-ids? (into #{} (filter is-frame?) ids)
+            detach-comment (fn [comment-threads]
```

Comment:
This function looks like an alias, why not calling directly to `update-comment-thread-frame` ?